sdk/azidentity/certificate_empty.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN BLOCK-----
```

Comment:
Can you please put this file and the other ones in a `testdata` subdirectory.

Reply:
Done